### PR TITLE
LPS-79261 Rename badge to label and overwrite hasFocus method to fix behavior of removing a label

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js [hunk 1]
```diff
@@ -186,6 +186,18 @@ AUI.add(
 						return instance.get('value') || [];
 					},
 
+					hasFocus: function(node) {
+						var instance = this;
+
+						var hasFocus = SelectField.superclass.hasFocus.apply(instance, arguments);
+						var nodeClass = '';
```
Comment (rafaprax) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js]: @alinedoleron Please remove the `nodeClass` unused variable. Thx

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js [hunk 2]
```diff
@@ -186,6 +186,18 @@ AUI.add(
...
+
+						if (node && node.hasClass('trigger-label-item-close')) {
+							hasFocus = true;
+						}
+						return hasFocus;
```
Comment (rafaprax) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js]: @alinedoleron Please add a new line before the line 198.

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by vicnate5** [comment]
Resent https://github.com/rafaprax/liferay-portal/pull/609

@alinedoleron Thanks for the tip on the locator. In the future, if you are comfortable, feel free to commit changes to the locators yourself :)

**Comment by vicnate5** [comment]
ci:close

**Pull request closed without merging**